scripts/components/Repo.js

```diff
@@ -1,19 +1,24 @@
 'use strict';
 
 import React, { PropTypes } from 'react';
-import PureRenderMixin from 'react/lib/ReactComponentWithPureRenderMixin';
+import shallowEqual from 'react/lib/shallowEqual';
 import { Link } from 'react-router';
 
-const Repo = React.createClass({
-  mixins: [PureRenderMixin],
+class Repo extends React.Component {
 
-  propTypes: {
-    repo: PropTypes.object.isRequired,
-    owner: PropTypes.object.isRequired
-  },
+  // TODO: make it either a HigherOrderComponent
```

Comment:
Sure. 

Besides for this example, do you suggest to avoid subclassing in general for such use cases? I'm currently refactoring one of my projects to replace `PureRenderMixin`...

I remember you also suggested it in your article ;)
https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750#4922